tests/cbmc/include/cbmc_proof/cbmc_utils.h

```diff
@@ -28,6 +30,27 @@ struct store_byte_from_buffer {
     uint8_t byte;
 };
 
+/**
+ * Asserts whether s2n_stuffer's fields are equivalent, except for read_cursor,
+ * which might change after a read operation.
+ */
+ void assert_stuffer_immutable_fields_after_read(const struct s2n_stuffer *lhs,
```

Comment:
nice

Reply:
You might want to note that this also checks that no byte within the blob changed.

Reply:
I updated the documentation.